release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -47,7 +47,191 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
 
 #### New Features
 
+###### Support for configure variables with records of records, record arrays and multi dimensional array types with Config.toml.
```

Comment:
###### Support for configurable variables with records having fields of record types